mevolib/align/__init__.py

```diff
@@ -132,3 +135,18 @@ def get_alignment(binary: str, infile: str, infile_format: str, args: str = 'def
             AlignIO.write(alignment, outfile_path, outfile_format)
         # Return the resultant alignment as a Bio.Align.MultipleSeqAligment object
         return alignment
+
+def main():
+    """Default call for Align module."""     
+    parser = argparse.ArgumentParser(
+        description="Performs the sequences alignment using the given alignment tool and arguments"
+    )
+    parser.add_argument("-t", "--tool", required=True, help="Alignment tool")
+    parser.add_argument("-i", "--input", required=True, help="FASTA file of unaligned sequences")
+    args = parser.parse_args()
+
+    # We split the input file to obtain its name
+    aux = args.input.split(".fasta")[0]    
```

Comment:
```suggestion
    filename = Path(args.input).stem
```
You may need to do `from pathlib import Path` above and replace `aux` below by `filename`.